site/_scss/site/objects/_footer.scss

```diff
@@ -1,3 +1,4 @@
+@import 'mixins';
```

Comment:
@amymanion don't import mixins to another partial; instead import it into `site/_scss/_styles.scss`